### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by kkovary** [datamol-io/molfeat#90] [Q&A] [answered]
I was curious what the release schedule is for `molfeat`, and what the process was for determining or asking for a new release.

**Comment by maclandrol** [comment, marked as answer]
Hi @kkovary, any changes or PR reviewed and merged would be followed by a release on both pypi and conda. We are usually releasing patch or minor versions (similar to [datamol](https://github.com/datamol-io/datamol)). 

If you believe the package (or some dependencies) is not up to date, please raise an issue.

**Comment by kkovary** [comment]
It looks like there have been several merges in since the last release (`0.10.1` on August 14, 2024). It would be great if the team was able to publish another release in order to gain access to some of these new features/patches. Thanks!
